frontend/integration-tests/tests/secrets.scenario.ts

```diff
@@ -1,64 +1,119 @@
-import { $, element, browser, by, ExpectedConditions as until } from 'protractor';
+/*eslint max-nested-callbacks: ["error", 5]*/
```

Comment:
You might just ignore the error in this file instead of trying to set it a different value